common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -293,35 +313,32 @@ private boolean fetchFromScheduledTaskQueue() {
             return true;
         }
         long nanoTime = AbstractScheduledEventExecutor.nanoTime();
-        Runnable scheduledTask = pollScheduledTask(scheduledTaskQueue, nanoTime, true);
-        while (scheduledTask != null) {
+        for (Runnable scheduledTask; (scheduledTask = pollScheduledTask(nanoTime)) != null;) {
```

Comment:
imho it would be easier to read as:

```java
for (;;) {
    Runnable scheduledTask = pollScheduledTask(nanoTime);
    if (scheduledTask == null) {
        break;
    }
    ....
}
```